### PR TITLE
Use squared spinner for active pipelines

### PR DESCRIPTION
**Pull request by xoxys**
Fixes: https://github.com/woodpecker-ci/woodpecker/discussions/2162#discussioncomment-8205012

- Use squared spinner to have a bit more space
- Display "9+" if more than 9 pipelines are running

https://github.com/user-attachments/assets/3f6c3836-f7c0-429f-ab76-cc8a1d22856f

### CODE REVIEW

File: web/src/components/layout/header/ActivePipelines.vue [hunk 1]
```diff
@@ -1,58 +1,57 @@
...
-      <div class="spinner-ring ring4" />
-    </div>
+    <div v-if="activePipelines.length > 0" class="spinner" />
     <div
-      class="flex items-center justify-center h-full w-full font-bold bg-white bg-opacity-15 dark:bg-black dark:bg-opacity-10 rounded-md"
+      class="z-1 flex items-center justify-center h-full w-full font-bold bg-white bg-opacity-15 dark:bg-black dark:bg-opacity-10 rounded-md"
     >
-      {{ activePipelines.length || 0 }}
+      <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+      {{ pipelineCount > 9 ? '9+' : pipelineCount }}
```
Comment (6543) [inline web/src/components/layout/header/ActivePipelines.vue:8]: the option to know the exact nuber would be still nice ...

what about a mouse hover over to show the number?

Comment (xoxys) [reply web/src/components/layout/header/ActivePipelines.vue:8]: Good idea will add it

Comment (xoxys) [reply web/src/components/layout/header/ActivePipelines.vue:8]: Good idea will add it

Comment (xoxys) [reply web/src/components/layout/header/ActivePipelines.vue:12]: image

**Review by 6543 — APPROVED** [review]
nice!!!

**Comment by xoxys** [comment]
Thanks. Will look into it

**Comment by xoxys** [comment]
Sure I can adjust it I also dont have a strong opinion on that.

**Pull request merged**